examples/ribot/BotLoader.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const RiveScript = require("riveScript");
+const path = require("path");
+const BotLoader = {};
+
+// Load a directory full of RiveScript documents (.rive files)
+const brainsDir = path.join(__dirname, "scripts");
+
+
+BotLoader.reset = function() {
+    console.log("loading rivescripts brain");
+    BotLoader.brain = new RiveScript();
+    BotLoader.brain.loadDirectory(brainsDir, loading_done, loading_error);
+}
+
+// Load an individual file.
```

Comment:
I can throw an eslint into the tree for the stuff I was working on but my preferred settings don't seem to match yours.